### PR TITLE
Merge master release/5.0.1xx preview8

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

File: eng/Version.Details.xml.orig [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    </Dependency>
+    <Dependency Name="Microsoft.Build" Version="16.8.0-preview-20371-01">
+      <Uri>https://github.com/microsoft/msbuild</Uri>
+      <Sha>e8338f5ad91d42c3196218648859e9e837099842</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Build.Localization" Version="16.8.0-preview-20371-01">
+      <Uri>https://github.com/microsoft/msbuild</Uri>
+      <Sha>e8338f5ad91d42c3196218648859e9e837099842</Sha>
+    </Dependency>
+<<<<<<< HEAD
```
Comment (sfoslund) [inline eng/Version.Details.xml.orig:44]: Merge issues

File: .vsts-ci.yml [hunk 2]
```diff
@@ -43,34 +43,41 @@ stages:
           queue: BuildPool.Windows.10.Amd64.VS2019.Pre.Open
         ${{ if ne(variables['System.TeamProject'], 'public') }}:
           name: NetCoreInternal-Pool
-          queue: buildpool.windows.10.amd64.vs2017
```
Comment (sfoslund) [inline .vsts-ci.yml:46]: Do we need the changes in this file?

**Review by sfoslund — CHANGES_REQUESTED** [review]
@wli3 Seems like there are a lot of merge issues. I put a PR out to merge preview 8 to master on Friday, was there something missing on that one? https://github.com/dotnet/sdk/pull/12632

File: .vsts-ci.yml (hunk 2 above)
Comment (wli3) [reply .vsts-ci.yml:46]: yes. This is not something arcade can automatically handle

**Comment by wli3** [comment]
@sfoslund i did not see #12632. Closing this PR

**Pull request closed without merging**